### PR TITLE
Add an icon to the Flutter Material Gallery

### PR DESCRIPTION
**Pull request by eseidelGoogle**
@abarth @hixie

### CODE REVIEW

**Comment by eseidelGoogle** [comment]
This also now includes a non-material-design-icons asset, as an example for https://github.com/flutter/flutter/issues/976

File: examples/material_gallery/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+Demo app for the Material Design widgets implemented in Flutter.
+
+## Icon
+
+Icon was generated using Android Asset Studio:
+https://romannurik.github.io/AndroidAssetStudio/icons-launcher.html#foreground.type=image&foreground.space.trim=0&foreground.space.pad=0.1&foreColor=607d8b%2C0&crop=0&backgroundShape=square&backColor=ffffff%2C100&effects=none
+
+From the Flutter Logo:
+https://github.com/flutter/flutter.github.io/blob/master/_includes/logo.html
+which appears to be CC by 4.0.
```
Comment (Hixie) [inline examples/material_gallery/README.md]: CC-BY

**Comment by Hixie** [comment]
LGTM.

FWIW, you have a shadow in the icon, which I think violates iOS guidelines. Not sure about Android.

**Comment by eseidelGoogle** [comment]
The icon is currently Android-only

File: examples/material_gallery/README.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+Demo app for the Material Design widgets implemented in Flutter.
+
+## Icon
+
+Icon was generated using Android Asset Studio:
+https://romannurik.github.io/AndroidAssetStudio/icons-launcher.html#foreground.type=image&foreground.space.trim=0&foreground.space.pad=0.1&foreColor=607d8b%2C0&crop=0&backgroundShape=square&backColor=ffffff%2C100&effects=none
+
+From the Flutter Logo:
+https://github.com/flutter/flutter.github.io/blob/master/_includes/logo.html
+which appears to be CC-BY 4.0.
```
Comment (abarth) [inline examples/material_gallery/README.md]: I created the Flutter logo myself.  We can license it however we like.

**Pull request merged**